components/MediaPlayer.vue

```diff
@@ -1,99 +1,247 @@
+            </span>
+            <span
+              v-if="currentTrack?.meta?.artist && currentTrack?.meta?.album"
+            >
+              -
+            </span>
+            <span
+              v-if="currentTrack?.meta?.album"
+              class="truncate text-base leading-snug text-label-2 dark:text-label-dark-2"
+              :title="currentTrack?.meta?.album"
+            >
+              {{ currentTrack.meta?.album }}
+            </span>
+          </div>
+        </div>
+        <button
+          v-if="status === MediaPlayerStatus.Playing"
+          @click.stop="pause()"
+        >
           <span class="flex aspect-square w-12 justify-center align-middle">
             <NuxtIcon
-              name="icon.rewind.large"
+              name="icon.pause.large"
               class="text-3xl group-hover:text-4xl"
             />
           </span>
         </button>
-      </div>
-      <div v-if="hasPrevious" class="group select-none">
-        <button @click.stop="previous()">
+        <button
+          v-if="status !== MediaPlayerStatus.Playing"
+          @click.stop="play()"
+        >
           <span class="flex aspect-square w-12 justify-center align-middle">
-            <NuxtIcon
-              name="icon.previous.track.large"
-              class="text-3xl group-hover:text-4xl"
-            />
+            <NuxtIcon name="play" class="text-3xl group-hover:text-4xl" />
           </span>
         </button>
       </div>
-      <div class="flex min-w-0 flex-1 gap-3">
+      <svg
+        width="100%"
+        height="28"
+        viewBox="0 0 100% 28"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        class="width-full"
+      >
+        <rect y="10" width="100%" height="8" rx="4" fill="#F5F6F7" />
+        <rect
+          :width="(currentPosition / currentTrackDuration) * 100 + '%'"
+          height="8"
+          transform="translate(0 10)"
+          fill="#0D131A"
+          rx="4"
+        />
+        <rect
+          v-if="(currentPosition / currentTrackDuration) * 100 > 0.95"
+          :width="(currentPosition / currentTrackDuration) * 105 + '%'"
+          height="8"
+          transform="translate(1.8 10)"
+          fill="#0D131A"
+        />
+      </svg>
+    </div>
+  </transition>
+
+  <transition
+    enter-active-class="transition-all duration-200 ease-out"
+    enter-from-class="opacity-0 translate-y-2"
+    leave-active-class="transition-all duration-200 ease-out"
+    leave-to-class="opacity-0 translate-y-2"
+  >
+    <div
+      v-if="open"
+      class="shadow-player h-100 absolute bottom-5 right-5 flex flex-col overflow-auto rounded-2xl bg-white-1 p-4 pb-0"
+      @click.stop="open = !open"
+    >
+      <div class="flex justify-between pb-[17px]">
+        <div class="rounded-2xl border">
+          <NuxtIcon name="icon.minify" filled class="text-2xl" />
+        </div>
+        <!-- TODO: REPLACE WITH CURRENT Content language -->
```

Comment:
Do you here mean the track of the currently playing track? This should be `currentTrack.language`.

Reply:
Right, ok. That at least returns the language enum